docs/common_tasks.rst

```diff
@@ -51,6 +51,45 @@ Running the tests
 
      (venv)vagrant@precise32:~/treeherder$ flake8
 
+
+Look up credentials for a project
+---------------------------------
+
+To submit data to an existing project, you need corresponding OAuth
+ credentials, even if you're submitting to your local server.
```

Comment:
There's a stray space at the beginning of this line, which is messing a bit with formatting.